plugins/kn-image/cmd/deploy.go

```diff
@@ -0,0 +1,276 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/knative/client-contrib/plugins/kn-image/tekton"
+	servingclientset_v1alpha1 "github.com/knative/client/pkg/serving/v1alpha1"
+	serving_v1alpha1_api "github.com/knative/serving/pkg/apis/serving/v1alpha1"
```

Comment:
Move to v1

Reply:
Hi Max, 
I have moved to use the serving v1 and latest Tekton v1 api.

Thanks!